android/engine/src/test/java/org/smartregister/fhircore/engine/ui/questionnaire/QuestionnaireActivityTest.kt

```diff
@@ -441,6 +441,39 @@ class QuestionnaireActivityTest : ActivityRobolectricTest() {
     verify { spiedActivity.dismissSaveProcessing() }
   }
 
+  @Test
+  fun testQuestionnaireTypeEditShouldAppendEditPrefixInActionBarTitle() {
+    questionnaireActivity.questionnaireType = QuestionnaireType.EDIT
+    questionnaireActivity.questionnaireViewModel.questionnaireConfig =
+      QuestionnaireConfig("form", "title", "form-id")
+
+    questionnaireActivity.updateViews()
+
+    Assert.assertEquals("Edit title", questionnaireActivity.supportActionBar?.title)
```

Comment:
can you fetch the lhs using `getString`?

Reply:
Hmm, could you elaborate?